packages/ndk/lib/domain_layer/usecases/lnurl/lnurl.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:convert';
+
+import 'package:http/http.dart' as http;
+
+import '../../../shared/logger/logger.dart';
+import 'lnurl_response.dart';
+
+/// LN URL utilities
+abstract class Lnurl {
```

Comment:
why is this abstract? 
I see that all methods are static but maybe you had something in mind?

Reply:
Just that it doesn't keep state, just utilities methods so far. Can that still be a usecase? Not sure if later it will....

Reply:
you're right in that it should not be abstract and keep the http transport somehow inside